tests/unittests/Foundation/ReferenceFoundation/TestNSBundle.mm

```diff
@@ -97,68 +98,75 @@
 static NSString* _main = @"main";
 static NSString* _type = @"swift";
 
-static NSString* _setupPlayground() {
-    NSArray* _bundleResourceNames = @[ @"hello.world", @"goodbye.world", @"swift.org" ];
-
-    // Make sure the directory is uniquely named
-    // WINOBJC: test environment is different, '/tmp/' is not a top-level directory
-    // NSString* tempDir =
-    //     [[@"/tmp/TestFoundation_Playground_" stringByAppendingString:[NSUUID UUID].UUIDString] stringByAppendingString:@"/"];
-    NSString* tempDir =
-        [[@"./tmp/TestFoundation_Playground_" stringByAppendingString:[NSUUID UUID].UUIDString] stringByAppendingString:@"/"];
-
-    NSError* error = nil;
-    [[NSFileManager defaultManager] createDirectoryAtPath:tempDir withIntermediateDirectories:false attributes:nil error:&error];
-    if (error) {
-        TraceError(L"TestNSBundle", L"%hs", [[error description] UTF8String]);
-        return nil;
+class NSBundlePlayground : public ::testing::Test {
+public:
+    explicit NSBundlePlayground() : ::testing::Test() {
     }
 
-    // Make a flat bundle in the playground
-    auto bundlePath = [tempDir stringByAppendingString:_bundleName];
-    [[NSFileManager defaultManager] createDirectoryAtPath:bundlePath withIntermediateDirectories:false attributes:nil error:&error];
-    if (error) {
-        TraceError(L"TestNSBundle", L"%hs", [[error description] UTF8String]);
-        return nil;
-    }
-
-    // Put some resources in the bundle
-    for (NSString* n in _bundleResourceNames) {
-        [[NSFileManager defaultManager] createFileAtPath:[[bundlePath stringByAppendingString:@"/"] stringByAppendingString:n]
+protected:
+    virtual void SetUp() {
+        NSArray* _bundleResourceNames = @[ @"hello.world", @"goodbye.world", @"swift.org" ];
+
+        // Make sure the directory is uniquely named
+        // WINOBJC: test environment is different, '/tmp/' is not a top-level directory
+        // NSString* tempDir =
+        //     [[@"/tmp/TestFoundation_Playground_" stringByAppendingString:[NSUUID UUID].UUIDString] stringByAppendingString:@"/"];
+        NSString* tempDir =
+            [[@"./tmp_TestFoundation_Playground_" stringByAppendingString:[NSUUID UUID].UUIDString] stringByAppendingString:@"/"];
```

Comment:
>./tmp_TestFoundation_Playground_ [](start = 16, length = 32)

if the intent is for cleanup to be reliable, does it make sense for us to remove /tmp/ from our gitignore?